webapp/src/shared/components/Row/Row.tsx

```diff
@@ -0,0 +1,54 @@
+import Icon, { IconVariant, IconSize } from '../Icon/Icon';
+import './Row.css';
+import { SmallAvatar, AvatarProps } from '../Avatar/Avatar';
+import { TextVariant, TextWeight } from '../Text/Text';
+import { default as Text } from '../Text/Text';
+
+type RowProps = {
+  iconVariant?: IconVariant;
+  onClick?: () => void;
+  avProps?: AvatarProps;
+  children?: string[];
+};
+
+export default function Row({
+  iconVariant,
+  onClick,
+  avProps,
+  children,
+}: RowProps) {
+  return (
+    <button className={`row paragraph-regular`} onClick={onClick}>
+      <div className={`row_wrapper`}>
+        {avProps && <SmallAvatar {...avProps} />}
+        {children && (
+          <div className={`row_text_wrapper`}>
+            {children[0] && (
+              <Text
+                variant={TextVariant.PARAGRAPH}
+                weight={TextWeight.MEDIUM}
+                parent_class="row_text_wrapper"
+              >
+                {children[0]}
+              </Text>
+            )}
+            {children[1] && (
+              <Text
+                variant={TextVariant.CAPTION}
```

Comment:
I think subtitle is not a caption but a regular paragraph, please review text sizes/types from Figma -> https://www.figma.com/file/efdjuSsiJqDyZbOxs9UTI7/App?node-id=159%3A32

Reply:
Yes, I understand that, but it looks bad. Maybe we have to wait to have the text sizes fixed?

Reply:
Also, plain no.
We agreed on figma because it looks good. Title is set to 'subheading', but the size of the font as displayed does not match what we have set for subheading. I have matched the text sizes, that looks to me like it's what we agreed on, not the text styles as we have them described in the top left corner. If there are inconsistencies, the displayed value should be prioritized.

![Screenshot from 2022-07-17 23-10-17](https://user-images.githubusercontent.com/2774420/179423272-c8c89c0b-a60a-48d1-ab30-d5bf4b367690.png)

![Screenshot from 2022-07-17 23-10-04](https://user-images.githubusercontent.com/2774420/179423309-e91919ca-ca0a-488a-8685-85eb8cc7ef33.png)

![Screenshot from 2022-07-17 23-09-46](https://user-images.githubusercontent.com/2774420/179423311-6198dbdc-eedb-4708-a782-4b63a1731ffa.png)

Reply:
Thanks for updating this :grinning:  We have indeed a task to revisit font sizes -> https://github.com/42AGV/ft_transcendence/issues/101 because, as you say, there are some sizes which look bad. But I think naming would still be the same due to what it represents.